mitosheet/mitosheet/enterprise/mito_config.py

```diff
@@ -13,9 +13,33 @@
 # the environment variables from previous mito_config_versions.
 MITO_CONFIG_KEY_VERSION = 'MITO_CONFIG_VERSION'
 MITO_CONFIG_KEY_SUPPORT_EMAIL = 'MITO_CONFIG_SUPPORT_EMAIL'
+MITO_CONFIG_KEY_CODE_SNIPPETS_SUPPORT_EMAIL = 'MITO_CONFIG_CODE_SNIPPETS_SUPPORT_EMAIL'
```

Comment:
It feels wrong we don't include the other keys we just added here. I think we probably should include these keys here (and access them through here) - we want all the configuration to be managed through a single location.

Since the code snippets URL and version should always be either both there or not both there, I could see the accessor for them returning them both together or neither (aka, we can enforce that in the type system). 

I think you'll have to change up how you then access the config, where you pass it to the steps manager, so you can pass it through to the API. That seems very sensible to me.